### PR TITLE
addheadernavelements

### PR DESCRIPTION
**Pull request by othukeodoko**
@michaella23 I added the header and the navigation elements as created in the issue linking them to each of the html tabs and also applying styling as seen in the design file.

### CODE REVIEW

**Review by BuckyBuck135 — COMMENTED** [review]
Hi @othukeodoko,
it looks great, well done!
I'll leave it to @michaella23 to review, but the idea I had behind the `contribute` "button" is that it would link to this repo. As such, it should rightfully be an `<a>` element rather than a `button`.

Geoffrey

**Comment by othukeodoko** [comment]
@BuckyBuck135 Thank you so much for providing clarity on this. As it wasn't clear where it should go. I will close the pull request, make the adjustment and resend.

**Comment by othukeodoko** [comment]
@BuckyBuck135 I have made the changes to link to the project repo. @michaella23 Kindly review also.

File: index.css [hunk 1]
```diff
@@ -14,7 +14,60 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.logo {
```
Comment (michaella23) [inline index.css:28]: You don't need to change it, but you can use a shorthand for margin.
If you have four different values, you can write it like this:
`margin: <top> <right> <bottom> <left>`
Notice there are no commas.
So in this case you could write `margin: 10px 40px 10px 10px`
If your right and left margin are the same, but top and bottom are different you can write it like this:
`margin: <top> <right and left> <bottom>`
If top and bottom are the same value and right and left are the same value (different from top and bottom), the shorthand uses two values:
`margin: <top and bottom> <right and left>`
And if the margin is the same on all four sides, you just define margin with that one value.

File: index.css [hunk 2]
```diff
@@ -14,7 +14,60 @@
...
+}
+.nav-items{
+  text-decoration: none;
+  margin-right: 20px;
+}
+.nav-items a {
+  text-decoration: none;
+  color: unset;
+}
+.nav-items .active{
```
Comment (michaella23) [inline index.css:55]: Thanks for adding `hover` and `active` states for anchor tags!

**Review by michaella23 — APPROVED** [review]
Great job! This code is clean and well-formatted, and the desktop design is looking good.

**Pull request merged**